test/different_headline_format_input.yaml

```diff
@@ -0,0 +1,18 @@
+REPO_RELEASE_URL: https://example.com/example-repo/releases/v1.0.0
+RELEASE_DATE: '2023-08-14T13:12:36Z'
+TAG_NAME: v0.18.2
+REPO_NAME: example-repo
+REPO_URL: https://example.com/example-repo
+CHANGELOG_BODY: |
+  ### [0.18.0](https://github.com/indykite/test-ik-jira/compare/v0.17.0...v0.18.0) (2023-08-14)
+
```

Comment:
Je to sice nedulezite, ale docela me to matlo, ze tady mas tag name `v0.18.2` ale changelog body obsahuje nadpis `[0.18.0]`. Nemelo by to byt stejne?

Reply:
Changelog generuje GitHub, ten neovlivnim. Ja pouzivam ${{ github.event.release.tag_name }}, kde to je vcetne toho `v`.